src/repositories/StakeRequestRepository.ts

```diff
@@ -128,6 +130,13 @@ export default class StakeRequestRepository extends Subject<StakeRequest> {
           type: DataTypes.STRING,
           allowNull: false,
         },
+        blockNumber: {
```

Comment:
this shouldn't be null. Every stake request will have a block number.

Reply:
Changes done!